src/content/docs/zh-cn/reference/errors/get-static-paths-removed-rsshelper.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: getStaticPaths RSS helper is not available anymore.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
+
+
+> **GetStaticPathsRemovedRSSHelper**: RSS助手已从 `getStaticPaths` 中删除。请改用新的 @astrojs/rss 包。 (E03012)
+
+## 哪里出了问题？
+`getStaticPaths` 不再提供生成 RSS 摘要的助手函数。 我们建议迁移到 [@astrojs/rss](/zh-cn/guides/rss/#设置-astrojsrss) 集成。
+
+**请参阅：**
+-  [RSS Guide](/zh-cn/guides/rss/)
```

Comment:
Pointing out this link might be untranslated, let me know if it's expected for "Guide" to not be translated here.

Reply:
I translated this https://github.com/withastro/docs/pull/2995

Reply:
@Yan-Thomas @liruifengv I changed the link translation to match the page title in the updated RSS guide, see #2995